test/t8_cmesh_generator/t8_cmesh_parametrized_examples/t8_cmesh_new_empty.hxx

```diff
@@ -0,0 +1,58 @@
+/*
+This file is part of t8code.
+t8code is a C library to manage a collection (a forest) of multiple
+connected adaptive space-trees of general element classes in parallel.
+
+Copyright (C) 2024 the developers
+
+t8code is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+t8code is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with t8code; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+#ifndef T8_CMESH_NEW_EMPTY_HXX
+#define T8_CMESH_NEW_EMPTY_HXX
+
+#include "test/t8_cmesh_generator/t8_gtest_cmesh_cartestian_product.hxx"
```

Comment:
```suggestion
#include <test/t8_cmesh_generator/t8_gtest_cmesh_cartestian_product.hxx>
```

Reply:
I'm not completely sure what the difference of "" and <> is, but a stack overflow suggested to keep it constant within a file

Reply:
This answer from StackOverflow might help: 
https://stackoverflow.com/a/21594